src/lib/bootloader/stage1.rb

```diff
@@ -158,20 +160,20 @@ def available_locations
     end
 
     def can_use_boot?
-      tm = Yast::Storage.GetTargetMap
       partition = Yast::BootStorage.BootPartitionDevice
 
-      part = Yast::Storage.GetPartition(tm, partition)
+      parts = partitions.with(name: partition)
```

Comment:
Of course it should.... in the future. We are just making sure the current code keep working. In the future a bigger refactoring to take advantage of the new functionalities in the library would be nice. No more passing strings but objects.. But that's phase II. :wink: